### PR TITLE
Upgrade samples to MSAL Angular v2

### PR DESCRIPTION
**Pull request by derisen**
This PR:

* Replaces Angular 9 SPA projects using msal-angular-v1 in chapter 1 and 2 with Angular 11 SPA projects using msal-angular-v2.
* Revises the READMEs accordingly as the samples are now using auth code flow with PKCE instead of implicit flow.

@mmacy chapter1 sample is using app roles and the [instructions](https://github.com/Azure-Samples/ms-identity-javascript-angular-spa-dotnetcore-webapi-roles-groups/blob/4733bc42f7ba0469777fc886e5e0b0effb31f3ff/chapter1/README.md#define-application-roles-1) taken from [How to: Add app roles to your application and receive them in the token](https://docs.microsoft.com/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps)

### CODE REVIEW

File: chapter1/AppCreationScripts/sample.json [hunk 1]
```diff
@@ -90,19 +89,19 @@
       "SettingFile": "\\..\\TodoListSPA\\src\\app\\app-config.json",
```
Comment (Shama-K) [inline chapter1/AppCreationScripts/sample.json:89]: File name is "auth-config.json".

Comment (derisen) [reply chapter1/AppCreationScripts/sample.json]: Done!

**Review by idg-sam — COMMENTED** [review]
Chapter 1 looks good, works as expected, need app role assignment steps for using app roles blade.

File: chapter1/README.md [hunk 2]
```diff
@@ -109,86 +111,130 @@ There are two projects in this sample. Each needs to be registered separately in
...
+1. Still on the same app registration, select the **App roles** blade to the left.
+1. Select **Create app role**:
+    - For **Display name**, enter a suitable name, for instance **TaskAdmin**.
+    - For **Allowed member types**, choose **User**.
+    - For **Value**, enter **TaskAdmin**.
+    - For **Description**, enter **Admins can read any user's todo list**.
+1. Select **Create app role**:
+    - For **Display name**, enter a suitable name, for instance **TaskUsers**.
+    - For **Allowed member types**, choose **User**.
+    - For **Value**, enter **TaskUsers**.
```
Comment (Shama-K) [inline chapter1/README.md:209]: Value should be **TaskUser** as per SPA config file.

File: chapter1/AppCreationScripts/sample.json [hunk 3]
```diff
@@ -58,6 +57,7 @@
         }
```
Comment (Shama-K) [inline chapter1/AppCreationScripts/sample.json:57]: Value should be **TaskUser** as per SPA config file.

Comment (derisen) [reply chapter1/AppCreationScripts/sample.json:57]: done!

File: chapter1/README.md (hunk 2 above)
Comment (derisen) [reply chapter1/README.md]: done!

**Review by idg-sam — APPROVED** [review]
Chapter 2 looks good as well. Great work!
Have made comments during live code review for minor issue (first click results in no permissions)

**Pull request merged**